pkg/csi/service/common/constants.go

```diff
@@ -162,16 +162,24 @@ const (
 	// IPs is Client IP address, IP range or IP subnet
 	IPs string = "ips"
 
-	// CSINamespace is the namespace of pvCSI in TKC Cluster
-	CSINamespace = "vmware-system-csi"
+	// CSINamespaceVanillaK8S is the namespace of CSI driver in Vanilla K8S
+	CSINamespaceVanillaK8S = "kube-system"
+
+	// CSINamespaceSVCluster is the namespace of CSI in Supervisor
+	CSINamespaceSVCluster = "vmware-system-csi"
+
+	// CSINamespaceTkgCluster is the namespace of CSI in TKG Cluster
+	CSINamespaceTkgCluster = "vmware-system-csi"
 
 	// CSIFeatureStatesConfigMapName is the name of configmap to store FSS values
 	CSIFeatureStatesConfigMapName = "csi-feature-states"
```

Comment:
It is too late, but ideally, all vSphere related configmap, secret, or any other driver resources we create, lets start prefixing them with vsphere. `vsphere-csi-feature-states` would be good.

Reply:
As of now there is no conflict. Not making further changes to address this comment